doc/API-Friendica.md

```diff
@@ -24,6 +24,28 @@ Returns a list of [Event](help/API-Entities#Event) entities for the current logg
 - `since_id`: (optional) minimum event id for pagination
 - `count`: maximum number of items returned, default 20
 
+### POST api/friendica/event_create
+
+Create a new event for the current logged in user.
+
+#### Parameters
+
+- `id` : (optional) id of event, event will be amended if supplied
+- `name` : name of the event (required)
+- `startTime` : start of the event (ISO), required
+- `endTime` : (optional) end of the event, event is open end, if not supplied
+- `desc` : (optional) description of the event
+- `place` : (optional) location of the event
+- `publish` : (optional) create message for event
+
+### POST api/friendica/event_delete
```

Comment:
Question is, if we want to use some more HTTP magic. See for example this part of the Mastodon API:
```
'/statuses/{id:\d+}'                 => [Module\Api\Mastodon\Statuses::class,                 [R::GET, R::PUT, R::DELETE]],
```
So with `GET` you get the event, with `DELETE` you delete it, ... Means: The endpoint stays the same, for example `api/friendica/event/{id:\d+}` and you can both fetch and delete it from there.

In my opinion we should use a `POST` to the `/event` endpoint to create a new post, a `GET` to `/event/id` to get it and a `DELETE` to `/event/id` to delete it - and possibly a `PUT` to edit it.

This is much cleaner I think.

Reply:
You would need Marco to pitch in on this topic, either approach is fine on its own, so it's a matter of convention, and Marco is the ultimate arbiter in this particular case.

Reply:
Given the difficulty to reach Marco, I wouldn't make any sweeping changes to the API. We don't use this REST model anywhere else, I'm not sure we should start now.

Reply:
So, this can be merged?

Reply:
I just got the green light by Marco: https://friendica.mrpetovan.com/display/3e98eba8-9663-2c6e-0358-e7d511152871

We won't go forward with the proposed structure changes at the moment.

Reply:
Yes it can, @annando suggested a complete change of the API structure but Marco would rather this PR (without the suggested changes) to be merged now.

Reply:
I really would like to unify the API concerning the whole wording. I guess that in some upcoming version I guess that I will deprecate all the old parameters and API calls.

Reply:
Sure, knock yourself out, but please merge this PR first.